sources/europe/gb/NLS-OSTownPlan-Edinburgh1940-1960.geojson

```diff
@@ -0,0 +1,136 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "id": "NLS_edinburgh1940_1960",
+        "attribution": {
+          "url": "http://maps.nls.uk/os/national-grid/index.html",
+          "text": "National Library of Scotland - Edinburgh 1940s-1960s",
+          "required": true
+        },
+        "name": "OS Town Plan, Edinburgh 1940s-1960s (NLS)",
+        "end_date": 1960,
+        "url": "http://geo.nls.uk/mapdata3/os/edinburgh_1250_out/{zoom}/{x}/{y}.png",
+        "max_zoom": 20,
+        "min_zoom": 13,
+        "country_code": "GB",
+        "icon": "http://nls.tileserver.com/nls70-nq8.png",
+        "type": "tms",
+        "start_date": 1940,
+        "description": "Detailed town plan of Edinburgh 1940s-1960s, courtesy of National Library of Scotland."
+      },
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              -3.2360482489647966,
+              55.89151111353117
+            ],
+            [
+              -3.3046531677245983,
+              55.89572134543474
+            ],
+            [
+              -3.3149528503417858,
+              55.92718255238699
+            ],
+            [
+              -3.322162628173817,
+              55.95025710655518
+            ],
+            [
+              -3.3204460144042858,
+              55.963134420287204
+            ],
+            [
+              -3.2983016967773326,
+              55.985707426430935
+            ],
+            [
+              -3.239441668964796,
+              55.995759311319716
+            ],
+            [
+              -3.17401885986327,
+              55.99818896319235
+            ],
+            [
+              -3.108444213867177,
+              55.985995507321725
+            ],
+            [
+              -3.0730819702148326,
+              55.953909317255636
+            ],
+            [
+              -2.996692657470692,
+              55.96236574567562
+            ],
+            [
+              -2.9681968688964746,
+              55.93391405059106
+            ],
+            [
+              -2.964763641357412,
+              55.86770228729157
+            ],
+            [
+              -2.9857063293456934,
+              55.80388597450652
+            ],
+            [
+              -3.0483627319335707,
+              55.814979407011165
+            ],
+            [
+              -3.1812286376953014,
+              55.81362907119958
+            ],
+            [
+              -3.1934165954589844,
+              55.890908513131365
+            ],
+            [
+              -3.2360482489647966,
+              55.89151111353117
+            ]
+          ]
+        ]
+      }
+    },
+    {
+      "type": "Feature",
```

Comment:
What is this second polygon meant for? Should the two geometries be merged, maybe?